packages/opentelemetry-node/src/instrumentation/PluginLoader.ts

```diff
@@ -113,8 +113,16 @@ export class PluginLoader {
         // Expecting a plugin from module;
         try {
           const plugin: Plugin = require(modulePath).plugin;
-
           if (!utils.isSupportedVersion(version, plugin.supportedVersions)) {
+            this.logger.error(
```

Comment:
I believe this should be a warning not an error. An error implies something is broken, where a warning is just telling the user they may be doing something they should not.

Reply:
I would like some more input on that, i personnaly prefer to raise an error since the plugin wont work.

Reply:
cc @mayurkale22